docs/docker/README.md

```diff
@@ -0,0 +1,184 @@
+# Deploy with Docker
+
+## Prerequisites
+
+Docker provides a way to run applications securely isolated in a container, packaged with all its dependencies and libraries.
+To learn more on Docker, kindly refer here: [What is Docker?](https://www.docker.com/what-docker).
+
+Make sure that you have the latest version of Docker installed on your machine. [Get latest version](https://www.docker.com/products/overview#/install_the_platform)
+
+[Notes for non-Linux systems](notes-for-non-linux.md)
+
+## Usage instructions ##
+
+### Step 1 - Setup network ###
+Create a network in order for the cBioPortal container and mysql database to communicate.
+```
+docker network create cbio-net
+```
+
+### Step 2 - Run mysql with seed database ###
+Start a MySQL server. The command below stores the database in a folder named
+`/<path_to_save_mysql_db>/db_files/`. This should be an absolute path.
+
+```
+docker run -d --restart=always \
+  --name=cbioDB \
+  --net=cbio-net \
+  -e MYSQL_ROOT_PASSWORD='P@ssword1' \
+  -e MYSQL_USER=cbio \
+  -e MYSQL_PASSWORD='P@ssword1' \
+  -e MYSQL_DATABASE=cbioportal \
+  -v /<path_to_save_mysql_db>/db_files/:/var/lib/mysql/ \
+  mysql:5.7
+```
```

Comment:
can we connect to existing mysql server not in the cbio-net network? if so, how?

Reply:
I believe you can point the hostname in portal.properties to a MySQL instance running on another machine as usual, and there are also ways to set up a hybrid approach with a container that is not isolated from the host networking stack.

However, these docs (authored and maintained by The Hyve on [thehyve/cbioportal-docker](https://github.com/thehyve/cbioportal-docker/blob/v2.1.1%2Bbackport6012/README.md#step-2---run-mysql-with-seed-database) over the past several years) assume that most users who start deploying with Docker will not want to install a MySQL server without isolation, making it a more niche use-case.

I feel that it may be more constructive to review adding documentation for it as a separate feature. Do you think that it should block integrating these docs, @khzhu?

Reply:
@fedde-s, you might want to put one sentence or two to let users know how to connect to an existing database. We use cbioportal docker web image with AKS, but the sql servers (master/slave) are outside the AKS.

Reply:
added a few sentences